source/src/ble/cs_Characteristic.cpp

```diff
@@ -63,9 +63,7 @@ void CharacteristicBase::init(Service* svc) {
 	//! attribute value //
 	/////////////////////
 
-	_uuid.init();
-	ble_uuid_t uuid = _uuid;
-
+	const ble_uuid_t uuid = _uuid.getUuid();
```

Comment:
`_uuid.getUuid()` returns by reference, but the local variable `uuid` is assigned a copy nonetheless because its type is not a reference. Depending on usage, the following may be preferable.

```suggestion
	const ble_uuid_t& uuid = _uuid.getUuid();
```